tests/Integration/Behaviour/behat.yml

```diff
@@ -256,37 +256,47 @@ default:
                 - Tests\Integration\Behaviour\Features\Context\Domain\CmsPageFeatureContext
         cart_rule:
-            paths:
-                - "%paths.base%/Features/Scenario/CartRule"
-            contexts:
-                - Tests\Integration\Behaviour\Features\Context\AttributeFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\AttributeGroupFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\Carrier\CarrierFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\CustomerFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\Carrier\CarrierRangesFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\ZoneFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\CarrierFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\CommonFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\CurrencyFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\CustomerFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\CategoryFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\CartRule\AddCartRuleFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\CartRule\CartRuleAssertionFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\CartRule\DeleteCartRuleFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\CartRule\EditCartRuleFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\CartRule\SetCartRuleRestrictionsFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\CountryFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\CustomerGroupFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\ManufacturerFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\Product\AddProductFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\Product\Combination\CombinationListingFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\Product\Combination\GenerateCombinationFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\Domain\SupplierFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\LanguageFeatureContext
-                - Tests\Integration\Behaviour\Features\Context\ShopFeatureContext
-                - Tests\Integration\Behaviour\Features\Transform\CurrencyTransformContext
-                - Tests\Integration\Behaviour\Features\Transform\SharedStorageTransformContext
-                - Tests\Integration\Behaviour\Features\Transform\StringToBoolTransformContext
+          paths:
```

Comment:
Can you please keep the same tab rules in this file (tab = 4 spaces) It highlights some modifications that are not really relevant and it breaks the uniformized format of the whole file